index.html

```diff
@@ -163,7 +165,7 @@ <h1 class="gameCardTitle" id="games_genre_rank_2_title">Title #2</h1>
                                             </div>
                                         </div>
     
-                                        <div style="height: 75px; background-color: #C0C0C0;" class="podiumStep">2</div>
+                                        <div style="background-color: #C0C0C0;" class="podiumStep">2</div>

```

Comment:
The problem that there is no difference between the 1st, 2nd and third place, You have to let the size in the style here.